### DISCUSSION TITLE
Favorite / Pinning in explorer

### DISCUSSION THREAD

**Discussion by kholvoet** [aws/aws-toolkit-jetbrains#3591] [Feature requests]
It would be awesome, if in the AWS Toolkit Explorer, we could pin / favorite Indvidual items (like 3 of 1500 lambdas or 1 of 50+ rds instances) and either show only the favorites / pins or have a smart group that collects only those.

Alternatively, a search / filter to get to the item. Even just enabling Crtl-F would be a great benefit if there are a non-trivial number of objects.

**Comment by mrichards01** [comment]
I'm not sure how this would manifest but I agree there is a lot of pointing and clicking in the UI to get to what you truly need. Particularly, in some organisations where multiple resources between teams may be defined under the same account, it can be nice to be able to at least mark items as favourites to return to when needed
